### DISCUSSION TITLE
Polling "widget" in top navigation

### DISCUSSION THREAD

**Discussion by robinheidrich** [filamentphp/filament#13837] [Help]
### Package

Widgets

### Package Version

v3.2.97

### How can we help you?

Does anyone know if it is possible to make a kind of "widget" in the top navigation that also has polling? I want to display a player count next to the brand logo. Just as a normal navigation item.

I couldn't even figure out how to use the top navigation while also using the sidebar navigation.

**Comment by leandrocfe** [comment]
You can use a custom view in the brand logo
https://filamentphp.com/docs/3.x/panels/themes#adding-a-logo

You can use `wire:poll` inside the view

https://livewire.laravel.com/docs/wire-poll
```html
<div wire:poll>
    {{ $playerCount }}
</div>
```
